src/toil/lib/misc.py

```diff
@@ -10,11 +13,19 @@
 import socket
 from contextlib import contextmanager
 
+# can't do from 'toil import subprocess' as __init__.py import this module!
+if os.name == 'posix' and six.PY2:
```

Comment:
Aren't we py3 only now?

Reply:
this was done before py3 only; I am going to update and simplify